pycbc/inference/models/gaussian_noise.py

```diff
@@ -90,8 +90,9 @@ class GaussianNoise(BaseDataModel):
         values are the data (assumed to be unwhitened). The list of keys must
         match the waveform generator's detectors keys, and the epoch of every
         data set must be the same as the waveform generator's epoch.
-    f_lower : float
+    lower_frequenc_cutoff : float
```

Comment:
`frequenc` -> `frequency`

Also, call it `low_frequency_cutoff` rather than `lower`.